manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml

```diff
@@ -65,5 +52,7 @@ vars:
     kind: Deployment
     name: katib-ui
 
+commonLabels:
```

Comment:
This can be removed

Reply:
Are you sure? Kustomize sometimes uses this for the selectors of deployments or services.

Reply:
I think yes, check: https://github.com/kubeflow/katib/pull/1507#discussion_r605749416, we decided with @yanniszark, that we don't need this labels more.

Reply:
Let's just remove these labels.
If we require them in the future we will update the Kustomization file.

Reply:
@andreyvelich From what I saw all the selectors were setup properly so I removed this now.